### PR TITLE
Token fix

### PR DESCRIPTION
**Pull request by gislenius**

### CODE REVIEW

**Comment by wcandillon** [comment]
Looks fantastic however the commits from  PR #654 appear there. Which I guess is fine if PR #654 is merged first.

File: queries/public/api/report-elements.jq [hunk 1]
```diff
@@ -152,17 +152,18 @@ let $result :=
...
+      let $component as object := $components[$$.Archive eq $archive and $$.Role eq $role]
+      let $members as object* := $component.Concepts[]
+      let $archive as object := $archives[$$._id eq $archive]
+      let $entity as object := $entities[$$._id eq $archive.Entity]
+      return
+        if ($onlyNames)
+        then distinct-values($concepts.Name)
+        else
+          if($profile-name eq "sec")
+          then
```
Comment (dknochen) [inline queries/public/api/report-elements.jq]: is this change intended to be here? Could you just double check?

Comment (gislenius) [inline queries/public/api/report-elements.jq]: Yes. I took the opportunity to refactor commonalities away.

**Comment by dknochen** [comment]
@gislenius in the tests module we could now refactor and use the backend:url(...) function. That would remove redundant code.

**Comment by gislenius** [comment]
@dknochen ... and there you go :-D

**Comment by gislenius** [comment]
Thanks :-)
